frontend/src/utils/sortAdmissions.ts

```diff
@@ -0,0 +1,66 @@
+import { Application } from "src/types";
+
+interface ApplicationComparator {
+  compare(app1: Application, app2: Application): number;
+  description: string;
+}
+
+const CreatedAtComparatorDesc: ApplicationComparator = {
+  compare: (app1, app2) => {
+    const date1 = new Date(app1.created_at);
+    const date2 = new Date(app2.created_at);
+    return date1.getTime() - date2.getTime();
+  },
+  description: "Dato Opprettet Synkende",
+};
+
+const CreatedAtComparatorAsc: ApplicationComparator = {
+  compare: (app1, app2) => {
+    const date1 = new Date(app1.created_at);
+    const date2 = new Date(app2.created_at);
+    return date2.getTime() - date1.getTime();
+  },
+  description: "Dato Opprettet Stigende",
+};
+
```

Comment:
I also kind of think I would prefer to have the field sorted on and the direction of the sort as separte options. Like having one seletor for what to sort by, and one for the direction of the sort. Just makes more sense to me. But it's not important, it can be changed later on if we want it:)